### PR TITLE
Add org_unit,org_uid, org_name to account/user objects

### PR DESCRIPTION
**Pull request by Aniak5**
image

#557

### CODE REVIEW

**Review by mr-splunk — DISMISSED** [review]

File: dictionary.json [hunk 1]
```diff
@@ -2071,7 +2071,7 @@
     },
     "org_unit": {
       "caption": "Org Unit",
-      "description": "The name of the organizational unit, withiin an organization.  For example, Finance, IT, R&D",
+      "description": "The name of the organizational unit, within an organization.  For example, Finance, IT, R&D",
```
Comment (rroupski) [inline dictionary.json:2074]: nitpick: missing `.` end the end of the description.

**Review by rroupski — CHANGES_REQUESTED** [review]

File: dictionary.json (hunk 1 above)
Comment (Aniak5) [reply dictionary.json:2074]: Oops, good catch!

**Comment by rroupski** [comment]
Should we use the new organization object?

**Comment by Aniak5** [comment]
> Should we use the new organization object?

Yup, I will wait until @pagbabian-splunk PR is merged.

**Pull request closed without merging**